language/Portuguese_Brazil/strings.po

```diff
@@ -274,7 +279,7 @@ msgstr "%1$s %2$s ( %3$d / %4$d pacotes do software )"
 #: src/frontend/mame/ui/selsoft.cpp:707
 #, c-format
 msgid "System: \"%1$s\" software list "
-msgstr "Sistema: \"%1$s\" lista dos softwares "
+msgstr "Sistema: lista dos softwares \"%1$s\" "
```

Comment:
I’m still concerned about how this is going to appear.  You can try it by starting MAME without specifying a system on the command line so the system selection menu appears, then selecting a system that uses software lists, for example “Apple //e”.

With this as-is, the heading on the software selection menu will say “Sistema: lista dos softwares "Apple //e" ”.  To me, this suggests that it’s showing content of an "Apple //e" software list, when actually “Apple //e” is just the name of the selected system, and it’s actually showing the combined contents of the three attached software lists (Apple II cleanly cracked disks, Apple II original disks, and Apple II miscellaneous disks).

Can this be worded to make it clearer that it’s showing the selected system name?

Reply:
changed to something better.